sentry-types/src/protocol/session.rs

```diff
@@ -130,3 +130,44 @@ pub struct SessionUpdate<'a> {
     #[serde(rename = "attrs")]
     pub attributes: SessionAttributes<'a>,
 }
+
+#[allow(clippy::trivially_copy_pass_by_ref)]
+fn is_zero(val: &u32) -> bool {
+    *val == 0
+}
+
+/// An aggregation grouped by `started` and `distinct_id`.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct SessionAggregateItem {
```

Comment:
No, this is still work in progress, so we haven't published documentation on this yet. To your point, though, at least the current state can be documented @Swatinem, with an "unstable" or "preview" note.

Reply:
I have the docs prepared since quite some time: https://github.com/getsentry/develop/pull/136
I can update them with a NOTE saying that while this is supported by now in relay, its still in preview and severely limited.